src/common/SurgePatch.cpp

```diff
@@ -1221,6 +1240,26 @@ unsigned int SurgePatch::save_patch(void **data)
     return psize;
 }
 
+Parameter *SurgePatch::parameterFromStorageName(std::string stName)
```

Comment:
This should take a std::string_view instead of a std::string (the map can still be std::string, but the method parameter should be std::string_view).

Reply:
Keeping as 'string' for now.